### ISSUE TITLE
resp may be problematic in templates

### ISSUE THREAD

**Issue by Xe** [dom96/jester#74]
I have no idea how I managed to do this [here](https://github.com/Xe/PonyAPI):

```
DEBUG get /last_aired
DEBUG   200 OK {Content-Length: 16, Content-Type: {
    "episode":
    {
        "is_movie": false,
        "name": "Spice Up Your Life",
        "air_date": 1465659000,
        "episode": 12,
        "season": 6
    }
}}
```

**Comment by dom96** [comment]
I don't know how it ever could have worked. Your ``httpReply`` (https://github.com/Xe/PonyAPI/blob/master/ponyapi.nim#L58) template is calling this ``resp`` overload (https://github.com/dom96/jester/blob/master/jester.nim#L366).

**Comment by dom96** [comment]
btw putting ``resp`` in a template like that may be problematic, Jester's route macro might not be able to detect it and transform it appropriately.

**Comment by PMunch** [comment]
Duplicate of #114? Might be fixed in #127
